supported-version/src/versions/mage-os/individual.json

```
@@ -25,6 +25,20 @@
         "nginx": "nginx:1.22",
         "os": "ubuntu-latest",
         "release": "2023-10-11T00:00:00+0000",
+        "eol": "2024-05-21T00:00:00+0000"
+    },
+    "mage-os/project-community-edition:1.0.2": {
+        "magento": "mage-os/project-community-edition:1.0.2",
+        "php": 8.3,
+        "composer": "2.7.4",
+        "mysql": "mariadb:10.6",
+        "elasticsearch": "opensearchproject/opensearch:2.12.0",
+        "rabbitmq": "rabbitmq:3.12-management",
```

```suggestion
        "rabbitmq": "rabbitmq:3.13-management",
```

According to https://experienceleague.adobe.com/en/docs/commerce-operations/installation-guide/system-requirements, this should be 3.13, right?